### DISCUSSION TITLE
Hardware requirements

### DISCUSSION THREAD

**Discussion by Sugavanam** [OpenIDC/mod_auth_openidc#1018] [General]
Hello Team,
I'm using mod_auth_openid with reverse proxy. I  approximately have 800 applications, Is there any minimum memory specifications for oidc to be used with apache.

**Comment by zandbelt** [comment]
not really by default it would use only something in the range of 10Mb: only when you're using "shm" cache and increase `OIDCCacheShmMax` to large numbers (to accomodate for a large number of parallel users), things would change; it does not depend on the number of applications behind it though
